File: ui_tests/PlaywrightAutomation/Pages/ContactUsPage.cs
using Microsoft.Playwright;

namespace PlaywrightAutomation.Pages
{
    internal class ContactUsPage : BasePage, IWebContainer
    {
        public string Container => "//div[@id='contactUsForm']";

        public ILocator Title => Page.Locator(Container).Locator("//div[@class='title']");
        public ILocator Text => Page.Locator(Container).Locator("//span[@class='styled-success-modal-text']/parent::span");

        public ILocator AttachFileInput => Page.Locator(Container).Locator("//input[@type='file']");
        public ILocator AttachedFileName => Page.Locator(Container).Locator("//label[@for='file']/span");

        public ILocator ErrorMessageForAttachInput => Page.Locator(Container).Locator("//div[@class='invalid']");
    }
}

File: ui_tests_playwright/automation/pages/CareerSite/ContactUsPage.ts
import BasePage from '../../base/page/BasePage';

export default class ContactUsPage extends BasePage {
	public fileInput() {
		return this.page.locator("//input[@type = 'file']");
	}

	public fileAttachError() {
		return this.page.locator("//div[@class='invalid files']");
	}

	public successModalMessage() {
		return this.page.locator("//div[contains(@class,'MessageWrapper')]/div/span");
	}
}
